### ISSUE TITLE
Lynda download error 

### ISSUE THREAD

**Issue by ridwan47** [ytdl-org/youtube-dl#20906]
![Capture_2019-04-30_12-25-02](https://user-images.githubusercontent.com/12793555/56943850-26d53a80-6b43-11e9-8e12-b27551e00408.png)

👇👇👇👇👇
ERROR: None: Failed to parse JSON  (caused by ValueError('Expecting property name enclosed in double quotes: line 1 column 3 (char 2)',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

This error shows when I try to download this lynda tutorial. 👇
https://www.lynda.com/Adobe-XD-tutorials/Learning-Adobe-XD-2019-REVISION/5022353-2.html
The error shows at downloading this video👇
https://www.lynda.com/Adobe-XD-tutorials/Masking/5022353/3510055-4.html

The code I used👇👇
Youtube-dl.exe --cookies cookies.txt -o "Lynda.com - %(playlist)s/%(chapter_number)s - %(chapter)s/%(playlist_index)s - %(title)s.%(ext)s" https://www.lynda.com/Adobe-XD-tutorials/Masking/5022353/3510055-4.html  --playlist-start 1 --all-subs

====================
VERBOSE
====================

C:\L>Youtube-dl.exe --verbose --cookies cookies.txt -o "Lynda.com - %(playlist)s/%(chapter_number)s - %(chapter)s/%(playlist_index)s - %(title)s.%(ext)s" https://www.lynda.com/Adobe-XD-tutorials/Masking/5022353/3510055-4.html  --playlist-start 1 --all-subs
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', '--cookies', 'cookies.txt', '-o', 'Lynda.com - %(playlist)s/%(chapter_number)s - %(chapter)s/%(playlist_index)s - %(title)s.%(ext)s', 'https://www.lynda.com/Adobe-XD-tutorials/Masking/5022353/3510055-4.html', '--playlist-start', '1', '--all-subs']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2019.04.30
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.17763
[debug] exe versions: none
[debug] Proxy map: {}
[lynda] 3510055: Downloading video JSON
[lynda] 3510055: Checking 0-540 video format URL
[lynda] 3510055: Checking 0-720 video format URL
[lynda] 3510055: Checking 0-360 video format URL
[lynda] 3510055: Checking 1-540 video format URL
[lynda] 3510055: Checking 1-720 video format URL
[lynda] 3510055: Checking 1-360 video format URL
ERROR: None: Failed to parse JSON  (caused by ValueError('Expecting property name enclosed in double quotes: line 1 column 3 (char 2)',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpfeq7l785\build\youtube_dl\extractor\common.py", line 898, in _parse_json
  File "C:\Python\Python34\lib\json\__init__.py", line 318, in loads
  File "C:\Python\Python34\lib\json\decoder.py", line 343, in decode
  File "C:\Python\Python34\lib\json\decoder.py", line 359, in raw_decode
ValueError: Expecting property name enclosed in double quotes: line 1 column 3 (char 2)
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpfeq7l785\build\youtube_dl\extractor\common.py", line 898, in _parse_json
  File "C:\Python\Python34\lib\json\__init__.py", line 318, in loads
  File "C:\Python\Python34\lib\json\decoder.py", line 343, in decode
  File "C:\Python\Python34\lib\json\decoder.py", line 359, in raw_decode
ValueError: Expecting property name enclosed in double quotes: line 1 column 3 (char 2)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpfeq7l785\build\youtube_dl\YoutubeDL.py", line 796, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpfeq7l785\build\youtube_dl\extractor\common.py", line 529, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpfeq7l785\build\youtube_dl\extractor\lynda.py", line 217, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpfeq7l785\build\youtube_dl\extractor\common.py", line 2852, in extract_subtitles
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpfeq7l785\build\youtube_dl\extractor\lynda.py", line 250, in _get_subtitles
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpfeq7l785\build\youtube_dl\extractor\common.py", line 891, in _download_json
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpfeq7l785\build\youtube_dl\extractor\common.py", line 875, in _download_json_handle
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpfeq7l785\build\youtube_dl\extractor\common.py", line 902, in _parse_json
youtube_dl.utils.ExtractorError: None: Failed to parse JSON  (caused by ValueError('Expecting property name enclosed in double quotes: line 1 column 3 (char 2)',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

===============================================

Please fix it if you can. I'm using the latest 2019.04.30 version.

**Comment by ridwan47** [comment]
I figured out the problem. There's no subtitle/transcript for that particular video. So the download stops and that error appears. 

This error should be fixed.. so that even if there's no subtitle/transcript then downloading should continue, just skip the download of subtitle/transcript it it's not available.

**Comment by ridwan47** [comment]
Nevermind, I figured out the error. Just add " -ciw" (without quotes) at the end of the line. 

It's good to use a combination of -ciw when downloading playlists.

-i, --ignore-errors    Continue on download errors, for example to skip
                       unavailable videos in a playlist
-w, --no-overwrites    Do not overwrite files
-c, --continue         Force resume of partially downloaded files.
                       By default, youtube-dl will resume downloads if possible.

Final Code I used:
youtube-dl --cookies cookies.txt https://www.lynda.com/Adobe-XD-tutorials/Learning-Adobe-XD-2019-REVISION/5022353-2.html -o "Lynda.com - %(playlist)s/%(chapter_number)s - %(chapter)s/%(playlist_index)s - %(title)s.%(ext)s"   --playlist-start 1 --skip-download --write-sub -ciw
